internal/website/content/pages/runtimevar.md

```diff
@@ -1,27 +1,27 @@
 ---
-title: "Runtimevar"
+title: "Runtime Configuration"
```

Comment:
Thinking aloud about this... Do we want the names to be package/component names or more descriptive sentences? We use "blob" but not "storage", and "secrets" is also quite terse

Reply:
I'm mildly in favor of having the sidebar links be more descriptive and then in the page describe more of what they do (which is the direction this PR goes toward), but I can be convinced the other way. If we use package names directly, then we should lowercase the page titles to match the package names.

Reply:
But the others are not descriptive... "Server", "Blob", etc. 
Not sure about the casing, since a lowercase page title feels strange

Reply:
Sure! I am willing to change more names. 🙂

Reply:
I prefer to keep it as-is for now. The main page has a description for each api when linking to it, and each API page starts with a description. I feel that over time folks will use the side bar more for reference and will appreciate brief names that map to memory well. We can discuss this separately with the PMs/DevRel folks if you'd like?

Reply:
Done. Kept as-is.